### PR TITLE
5387: fix margin and created icon files

### PR DESCRIPTION
**Pull request by sinejespersen**
#### Link to issue

https://platform.dandigbib.org/issues/5387

#### Description

- added icons as .tsx-files (for color-purposes)
- sortByLoanDate to helpers.ts
- add spacing to align

#### Screenshot of the result

image

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

File: src/apps/loan-list/helpers.ts [hunk 1]
```diff
@@ -16,6 +16,14 @@ export const getAmountOfRenewableLoans = (list: LoanV2[]) => {
...
 
+export const sortByLoanDate = (list: LoanV2[]) => {
+  return list.sort(
+    (objA, objB) =>
+      new Date(objA.loanDetails.loanDate).getTime() -
+      new Date(objB.loanDetails.loanDate).getTime()
+  );
+};
+
 export const formatDate = (date: string) => {
```
Comment (spaceo) [inline src/apps/loan-list/helpers.ts:27]: Think I have mentioned it in another PR. But I think this function belongs to the global date helper file in core.

File: src/apps/loan-list/materials/stackable-material.tsx [hunk 2]
```diff
@@ -87,6 +87,7 @@ const StackableMaterial: React.FC<StackableMaterialProps> = ({
               </button>
             )}
           {materialIsOverdue(dueDate) && (
+            // todo 5387.2.2b.iv.3.b.i. Se Bilag 6 pkt. 2
             <a
```
Comment (spaceo) [inline src/apps/loan-list/materials/stackable-material.tsx:91]: Should be Link component but I get the point that it is a TODO :)

Comment (sinejespersen) [reply src/apps/loan-list/materials/stackable-material.tsx:91]: yep, it'll be fixed

File: src/apps/loan-list/helpers.ts (hunk 1 above)
Comment (sinejespersen) [reply src/apps/loan-list/helpers.ts:27]: I'll move it and merge

**Pull request merged**